carbon-design-system/carbon-addons-iot-react
# feat(overflowmenu): add support for button label

Closes #3714 

**Summary**

- Add prop to enable label text in `OverflowMenu` button

**Change List (commits, features, bugs, etc)**

- Add prop for `OverflowMenu` component
- Add i18n prop for `Table` component
- Update docs
- Update public API

**Acceptance Test (how to verify the PR)**

- Go to [this story](https://deploy-preview-3718--carbon-addons-iot-react.netlify.app/?path=/story/1-watson-iot-table--playground)
- In the "i18n" knob write some text in `batchActionsOverflowMenuText`
- In the "Selections & actions" knob enable `useRadioButtonSingleSelect` and reselect `hasRowSelection: multi`
- Select some row
- Verify that overflow button in batch actions has label

**Regression Test (how to make sure this PR doesn't break old functionality)**

- tests here



**Things to look for during review**
